.doks/doks-cluster/variables.tf

```diff
@@ -58,3 +62,42 @@ variable "cluster_name" {
   type = string
   default = "default"
 }
+
+######################
+# databases
+######################
+
+variable "postgres_version" {
+  description = "The postgres version to be used."
+  type = string
+  default = "10"
+}
+
+variable "postgres_default_admin" {
+  description = "The default admin user on digital ocean postgres."
+  type = string
+  default = "doadmin"
+}
+
+variable "db_names" {
+    type = list(string)
+    default = ["accounts", "analysis", "results", "metrics", "filestore", "jobs", "crow", "activities", "hotspots", "listener"]
+}
+
+variable "postgres_instance_type" {
+  description = "The instance type for the postgres clusters."
+  type = string
+  default = "db-s-4vcpu-8gb"
+}
+
+variable "connection_pool_size" {
+  description = "Number connections for each connection pool."
+  type        = number
+  default     = 17
```

Comment:
With the current db cluster size, this is the maximum number of connections we can have across all connection pools.
There is one edge case that I verified with the `engine` component that is important to keep in mind.
When rotating the deployment of the engine, the newly created pod fails to successfully connect to all dbs, due to insufficient available connections on the pools. This means that the deployment will be in a limbo state as the new `engine` pod never becomes ready.
This may happen with other components as they rotate, although I am not absolutely sure.
The easy solution is, of course, to bump up the instance type which will retroactively allow us to increase the conn. pool sizes. We just need to foot the bill in that case :)